techniques/general/G167.html

```diff
@@ -10,16 +10,33 @@
       <section class="example">
          <h3>A search function</h3>
          
-            <p>A Web page contains a text field where the user can enter search terms and a button labeled "Search" for performing the search. The button is positioned right after the text field so that it is clear to the user that the text field is where to enter the search term.</p>
+            <p>A web page contains a text field where the user can enter search terms and a button labeled "Search" for performing the search. The button is positioned right after the text field so that it is clear to the user that the text field is where to enter the search term.</p>
             <div>
             	<img alt="A text field with a button positioned to the right demonstrating the use of a button to label a field." src="img/button-as-label.jpg"/>
             </div>
          
       </section>
+
+      <section class="example">
+         <h3>A "reply" or "send message" field</h3>
+         <p>A chat or email web application will often present users a single or multi-line text field used for composing a message.  The field does not have a visible text label, but the purpose of the field - to compose a message to send - is indicated by it being paired with a button represented by a "send message" icon.</p>
```

Comment:
Similar as with the text button for 'Search' above, the icon here serves as a label. I suggest to clearly state that.

<p>A chat or email web application will often present users a single or multi-line text field used for composing a message. It is often paired with an icon button labeled with a 'send message' icon (usually a paper plane). The icon also serves as a label for the message field.</p>